.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,65 @@
+name: ci testing
+
+
+on:
+  push:
+    branches: master
+  pull_request:
+
+
+jobs:
+
+  run_test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [
+          3.7,
+          3.8,
+          3.9,
+        ]
+        include:
+          - python-version: 3.7
+            tox-env: py37
+          - python-version: 3.8
+            tox-env: py38
+          - python-version: 3.9
+            tox-env: py39
+
+    steps:
+
+      - name: cancel previous workflows that are still running
+        uses: styfle/cancel-workflow-action@0.8.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: checkout latest commit
+        uses: actions/checkout@v2
+
+      - name: setup python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: install python dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install tox
+
+      - name: linting and code style
+        if: matrix.python-version == '3.7'
+        run: tox -vv -e lint -- --color=yes
```

Comment:
`--color=yes` is actually a `pytest` command and we just use `tox` ability to send through the CLI options. So it does not really make sense to use it for nontest `tox` environments. I think it will be just ignored.

Reply:
You are right, I have to fix that!

IIUC, this is implemented by this line in `tox.ini`:
```bash
    pytest {posargs:tests}
```

Do we have reasons to prefer this over a specific section `[pytest]` in `tox.ini`?

Reply:
IMO having it inside of `.github/workflows/run-tests.yml` is better because for example on my laptop the colors work out of the box and it is just github actions that need this extra option.

Reply:
Fair enough, fixed in e52753a!